cmd/plugin.go

```diff
@@ -172,10 +176,15 @@ Example:
 	return cmd
 }
 
-// exitCode=1 For unknown errors resulting in panics
-// exitCode=2 For insufficient/wrong arguments passed in the command
-// exitCode=3 For errors related to loading state, loading version data or an issue contacting the update server.
-// exitCode=4 For plugin listing failures
+var pluginInstallSteps = []string{
+	"Downloading",
+	"Installing Plugin",
+	"Installing Docs",
+	"Installing Config",
+	"Updating Steampipe",
+	"Done",
+}
+
```

Comment:
can we just do this: 
```
func runPluginInstallCmd(cmd *cobra.Command, args []string) {
	ctx := cmd.Context()
	utils.LogTime("runPluginInstallCmd install")
	defer func() {
		utils.LogTime("runPluginInstallCmd end")
		if r := recover(); r != nil {
			utils.ShowError(ctx, helpers.ToError(r))
			exitCode = constants.ExitCodeUnknownErrorPanic
		}
	}()

	// args to 'plugin install' -- one or more plugins to install
	// plugin names can be simple names ('aws') for "standard" plugins,
	// or full refs to the OCI image (us-docker.pkg.dev/steampipe/plugin/turbot/aws:1.0.0)
	plugins := append([]string{}, args...)
	installReports := make(display.PluginInstallReports, 0, len(plugins))

	if len(plugins) == 0 {
		fmt.Println()
		utils.ShowError(ctx, fmt.Errorf("you need to provide at least one plugin to install"))
		fmt.Println()
		cmd.Help()
		fmt.Println()
		exitCode = constants.ExitCodeInsufficientOrWrongArguments
		return
	}

	// a leading blank line - since we always output multiple lines
	fmt.Println()

	statusSpinner := statushooks.NewStatusSpinner()
	progressBars := uiprogress.New()
	installWaitGroup := &sync.WaitGroup{}
	dataChannel := make(chan *display.PluginInstallReport, len(plugins))

	progressBars.Start()

	for _, pluginName := range plugins {
		installWaitGroup.Add(1)
		go doPluginInstall(ctx, progressBars, pluginName, dataChannel)
	}

	go func() {
		installWaitGroup.Wait()
		close(dataChannel)
	}()

	for report := <-dataChannel; report != nil; {
		installReports = append(installReports, report)
	}

	progressBars.Stop()

	statusSpinner.UpdateSpinnerMessage("Refreshing connections...")
	refreshConnectionsIfNecessary(ctx, installReports, true)
	statusSpinner.Done()
	
         display.PrintInstallReports(installReports, false)

	// a concluding blank line - since we always output multiple lines
	fmt.Println()
}
```